Assignment6.Rmd

```diff
@@ -48,11 +53,14 @@ post(c.tree1, file = "tree1.ps", title = "MOOC") #This creates a pdf image of th
 #If we are worried about overfitting we can remove nodes form our tree using the prune() command, setting cp to the CP value from the table that corresponds to the number of nodes we want the tree to terminate at. Let's set it to two nodes.
 
 ```{r}
-c.tree2 <- prune(c.tree1, cp = )#Set cp to the level at which you want the tree to end
+c.tree2 <- prune(c.tree1, cp=0.2)#Set cp to the level at which you want the tree to end
+#Question: is there any rule for setting the cp to a certain level? I chose to put "0.2" here just to get rid of the third layer.
```

Comment:
Good question! It depends on context. There will always be a trade-off between accuracy on the training set and generalizability. It depends on what the consequences are for choosing a less specifically accurate but more generalizable model. In the case of MOOC certification we would likely want a more general model as it is more important to us to drive certification than to ensure that any intervention we serve based on our model gets in front of the right person in most accurate way.